src/tools/perf/lib/libperf.c

```diff
@@ -81,7 +81,9 @@ static const char *perf_iface_ops[] = {
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_SHORT)]  = "tag eager short",
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_BCOPY)]  = "tag eager bcopy",
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_ZCOPY)]  = "tag eager zcopy",
-    [ucs_ilog2(UCT_IFACE_FLAG_TAG_RNDV_ZCOPY)]   = "tag rndv zcopy"
+    [ucs_ilog2(UCT_IFACE_FLAG_TAG_RNDV_ZCOPY)]   = "tag rndv zcopy",
+    [ucs_ilog2(UCT_IFACE_FLAG_EP_CHECK)]         = "ep check",
+    [ucs_ilog2(UCT_IFACE_FLAG_EP_KEEPALIVE)]     = "ep keepalive"
```

Comment:
this seems to be excessive, what is the scenario when it will be used?

Reply:
but we will not request `UCT_IFACE_FLAG_EP_CHECK` or `UCT_IFACE_FLAG_EP_CHECK` flags explicitly.

Reply:
this is general function to check capability, may be used in future to test performance when one of keepalive method is forced

Reply:
i think it's ok to make sure all flags would be present here, I don't see a downside to adding it